client/scripts/views/modals/link_new_address_modal.ts

```diff
@@ -68,18 +69,20 @@ const EthereumLinkAccountItem: m.Component<{
             jwt: app.user.jwt,
           });
 
-          if (result.exists) {
-            if (result.belongsToUser) {
-              notifyInfo('This address is already linked to your current account.');
-              vnode.state.linking = false;
-              return;
-            } else {
-              const modalMsg = 'This address is currently linked to another account. '
-                + 'Remove it from that account and transfer to yours?';
-              const confirmed = await confirmationModalWithText(modalMsg)();
-              if (!confirmed) {
+          if(!(app.chain && (app.chain as Token).isToken && (app.chain as Token).isUninitialized)) {
```

Comment:
Is the check for `.isUninitialized` not needed because we have to wait for `app.chain` to initialize first? or is it because there is a separate child process for connecting to the contract after connecting to the chain? if so, why are they not the same process?

Reply:
See above, `isUninitialized` is about whether the token chain has been created in the database